.eslintignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 coverage
 build
+dist
```

Comment:
In toolkit, we will change the webpack output to be ['dist'](https://github.com/cerner/terra-toolkit/pull/81/files#diff-c6cf2bceff376d6c36d8c70fc6db3bb1R17) when using the serve-static. When using this serve option, why are we choosing 'dist' instead of using 'build' like we defaulted to? I agree we should still set it for consumer that may use a different option, but should we align with ours?

Reply:
This ignore is un-needed. I was playing with switching to dist from build because dist is the webpack default, but decided it wasn't worth it. I can remove.